docs/development/release.md

```diff
@@ -72,59 +124,66 @@ Similar to #4, make sure the DATA_MODEL_CHANGELOG is updated if there are data m
 git add CONTRIBUTORS.md
 ```
 
+Commit and push these changes, if any.
```

Comment:
```suggestion
Commit and push these changes, if any.

```bash
git commit -m "Update CONTRIBUTORS.md"
git push release-MAJOR.MINOR.x
```

```

This is a nit